src/gui/gtkentry.c

```diff
@@ -169,78 +169,78 @@ void dt_gtkentry_setup_completion(GtkEntry *entry, const dt_gtkentry_completion_
 const dt_gtkentry_completion_spec *dt_gtkentry_get_default_path_compl_list()
 {
   static dt_gtkentry_completion_spec default_path_compl_list[]
-      = { { "ROLL_NAME", N_("$(ROLL_NAME) - roll of the input image") },
-          { "FILE_FOLDER", N_("$(FILE_FOLDER) - folder containing the input image") },
-          { "FILE_NAME", N_("$(FILE_NAME) - basename of the input image") },
-          { "FILE_EXTENSION", N_("$(FILE_EXTENSION) - extension of the input image") },
+      = { { "ROLL.NAME", N_("$(ROLL.NAME) - roll of the input image") },
+          { "FILE.FOLDER", N_("$(FILE.FOLDER) - folder containing the input image") },
+          { "FILE.NAME", N_("$(FILE.NAME) - basename of the input image") },
+          { "FILE.EXTENSION", N_("$(FILE.EXTENSION) - extension of the input image") },
           { "VERSION", N_("$(VERSION) - duplicate version") },
-          { "VERSION_IF_MULTI", N_("$(VERSION_IF_MULTI) - same as $(VERSION) but null string if only one version exists") },
-          { "VERSION_NAME", N_("$(VERSION_NAME) - version name from metadata") },
+          { "VERSION.IF.MULTI", N_("$(VERSION.IF.MULTI) - same as $(VERSION) but null string if only one version exists") },
```

Comment:
Just a quick review, this should be `VERSION.IF_MULTI` I think as IF MULTI are not classification but an single word to me.

Reply:
Right!

Reply:
done